packages/react-hooks/documentation/KeyListenerExample.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useRef } from "react";
+import { useKeyListener } from "../src/useKeyListener";
+
+export default function KeyListenerExample() {
+    const [pressedKeys, setPressedKeys] = useState<string[]>([]);
+    const keys = ["ArrowUp", "ArrowDown", "ArrowLeft", "ArrowRight"];
+    const ref = useRef(null);
+
+    function onKeyPressInside(event: KeyboardEvent) {
+        setPressedKeys([...pressedKeys, event.key]);
+    }
+    const resetList = () => setPressedKeys([]);
+    useKeyListener(ref, keys, onKeyPressInside);
+
+    return (
+        <section className="hooks-example key-listener-example jkl-spacing--bottom-3">
+            <p className="hooks-example__header">Lytter etter trykk på piltastene</p>
+            <p className="hooks-example__text">
+                Flytt fokus til knappen under med Tab eller ved å trykke på den. Trykk på piltastene vil nå listes opp
+                under. Klikk utenfor knappen, eller flytt fokus med Tab, for å se at tastetrykkene ikke lenger
+                registreres.
+            </p>
+            <button
+                ref={ref}
+                className="key-listener-example__spaced-item jkl-button jkl-button--secondary"
```

Comment:
Sette denne til tertiary for å matche de andre eksemplene?

Reply:
Valgte secondary for å gjøre det tydeligere at det er en knapp. Vet ikke om det er et ønsket mønster med kun tertiary? Men kan sikkert slenge på en "aktivér"-knapp spnn som på de andre, og så gjøre om denne 😊